docs/components/grapesjs.rst

```diff
@@ -0,0 +1,74 @@
+GrapesJS Builder
+################
+
+Currently, Mautic uses the GrapesJS Builder for Emails and Landing pages. 
+
+Setup 
+*****
+
+.. code-block:: BASH
+
+    npm install
+
+.. vale off
+
+Configure Babel, ESLint, Prettier
+=================================
+
+.. vale on
+
+Use the template files provided. For example .eslintrc.temp
+
+How to test standalone
```

Comment:
What does it mean to test standalone? This isn't very clear. Perhaps we could explain why you would do this and what it would allow you to do?